src/Symfony/Component/Serializer/Normalizer/DateTimeNormalizer.php

```diff
@@ -113,6 +113,8 @@ public function denormalize(mixed $data, string $type, string $format = null, ar
             if (false !== $object) {
                 return $object;
             }
+
+            trigger_deprecation('symfony/serializer', '6.2', 'Relying on a datetime constructor as a fallback when using a specific default date format (`datetime_format`) for the DateTimeNormalizer is deprecated. Respect the "%s" default format.', $defaultDateTimeFormat);
```

Comment:
What will the code look like in 7.0? Can we add a hint as a comment here?

Reply:
Depending on how we answer the questions in PR description, In 7.0, the fallback path will be either totally removed or disabled by default.

Reply:
I'd got with 3 when format is `auto`.